src/rest_framework_dso/pagination.py

```diff
@@ -137,10 +133,35 @@ def __init__(self, results_field=None):
     def get_paginated_response(self, data):
         """Wrap the data in all pagination parts."""
         # Add the _links, and add the HTTP headers.
+        self.request.accepted_renderer.setup_pagination(self)
```

Comment:
Als je dit hier doet is de `DelegatedPageNumberPagination` overbodig geworden. Wellicht wel goed om dat ook te verwerken, en dan die comments van die class hier op te nemen. In de docs van `DelegatedPageNumberPagination` staat ook het design probleem van DRF met betrekking tot pagination.

Reply:
de get_paginated_response() methods geven wel andere output. levert dat geen problemen op?

Reply:
ow I stand corrected. Wat zou kunnen doen is `DSOPageNumberPagination` laten overerven van `DelegatedPageNumberPagination`. In feite is dit dezelfde functionaliteit.